### PR TITLE
Add simplecov for tests

### PR DESCRIPTION
**Pull request by kubapol**

### CODE REVIEW

**Comment by igorkasyanchuk** [comment]
Please fix specs on Travis CI - https://travis-ci.org/igorkasyanchuk/rails_db/builds/101488279

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ end
 group :test do
   gem "sqlite3"
   gem "jquery-rails"
+  gem "simplecov", :require => false
```
Comment (igorkasyanchuk) [inline Gemfile]: add development dependency to gemspec

**Comment by kubapol** [comment]
Fixed.

File: rails_db.gemspec [hunk 2]
```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'simple_form'
   s.add_dependency 'ransack'
   s.add_dependency 'kaminari'
+  s.add_dependency 'simplecov'
```
Comment (igorkasyanchuk) [inline rails_db.gemspec]: please add it as "add_development_dependency" (see below in that file)

**Comment by kubapol** [comment]
Now, TravisCI build fails.

**Comment by igorkasyanchuk** [comment]
Maybe try to install on Travis CI this gem (something like)

https://docs.travis-ci.com/user/common-build-problems/#Newer-version-of-CocoaPods-required

So travis CI will install all gems from gemspec + simplecov separatelly and then just require simplecov in rails_spec.rb file

Main idea is to not include this gem as hard dependency to our gem

@bbenezech maybe you can suggest something?

**Comment by igorkasyanchuk** [comment]
Hi @kubapol, do you plan to try my suggestions?

**Comment by bbenezech** [comment]
@igorkasyanchuk what is the issue?
simple_cov is needed in development for a rake task?

**Comment by igorkasyanchuk** [comment]
issue is that we just need to edit Travis.yml file in this PR to pass specs in Travis CI
but in same time do not add this gem as mandatory dependency

**Comment by kubapol** [comment]
I'll try to fix it next week. I'm too busy right now.

**Pull request closed without merging**